bootstrap-cpp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+usage() {
+    echo "Usage: $0 <source-dir> <build-dir>" >&2
+}
+
+if [ $# -eq 2 ]; then
+    SOURCE_DIR=$(realpath $1)
+    BUILD_DIR=$(realpath $2)
+elif [[ $# -eq 1 && ( "$1" = "-h" || "$1" = "--help" )]]; then
+    usage
+    exit 0
+else
+    echo "Wrong number of arguments: expected 2, was $#" >&2
+    usage
+    exit 1
+fi
+
+mkdir -p "${BUILD_DIR}"
+
+source /opt/rh/devtoolset-6/enable || true
```

Comment:
/opt/rh/devtoolset-6/root/usr/lib64 is prepended to LD_LIBRARY_PATH, but we do not use any of the libs there. python path is also set, but we do not use this either. Conclusion: We can keep it as is for now.